# HG changeset patch
# User Rik <rik@octave.org>
# Date 1384315225 28800
#      Tue Nov 12 20:00:25 2013 -0800
# Node ID 730bc06134f9617979ff5c10bba36b99e5a52075
# Parent  01496d4811b19d3ba5c380b396bbf9ac67aca3a2
test: Don't run bug-38576.tst on Windows systems where ls does not exist.

* test/bug-38576.tst: Wrap test with "if (! ispc ())" block.

diff --git a/test/bug-38576.tst b/test/bug-38576.tst
--- a/test/bug-38576.tst
+++ b/test/bug-38576.tst
@@ -5,12 +5,16 @@
 %!function r = f2 ()
 %!  [u,ls,v] = svd (1);
 %!  r = eval ("ls -1;");
 %!endfunction
 %!function r = f3 (ls)
 %!  r = eval ("ls -1;");
 %!endfunction
 
-%!assert (f1 (), 0);
-%!assert (f2 (), 0);
-%!assert (ischar (f3 ()), true);
-%!assert (f3 (1), 0);
+%!test
+%! ## Windows systems can't run "ls -1"
+%! if (! ispc ())
+%!   assert (f1 (), 0);
+%!   assert (f2 (), 0);
+%!   assert (ischar (f3 ()), true);
+%!   assert (f3 (1), 0);
+%! endif
